components/tooltip/tooltip-help.js

```diff
@@ -0,0 +1,92 @@
+import '../colors/colors.js';
+import '../tooltip/tooltip.js';
+import { css, html, LitElement } from 'lit';
+import { bodySmallStyles } from '../typography/styles.js';
+import { FocusMixin } from '../../mixins/focus-mixin.js';
+import { FocusVisiblePolyfillMixin } from '../../mixins/focus-visible-polyfill-mixin.js';
+
+/**
+ * A component used to display additional information when users focus or hover on certain text.
```

Comment:
Perhaps I'm being too picky here, but do we need this to match 100% to what we say in the [readme/Daylight page](https://github.com/BrightspaceUI/core/pull/2526/files#diff-1c38718d67e5f3d531c46a45d0a1ebae23b035384623ebdc2eeca8f8e87a99b1R157)?